src/watchd/runtime.rs

```diff
@@ -85,8 +85,20 @@ impl WatcherRuntime {
         let msg = self.receiver.recv()?;
         debug!("Processing message {}", msg);
         trace!("Message details: {:?}", msg);
-        // TODO: Forward electrum notifications over the bridge
-        // self.send_over_bridge(msg.into()).expect("watcher bridge is halted");
+        // Forward electrum notifications over the bridge
+        match msg {
+            ElectrumUpdate::TxBatch(transactions, _) => {
+                for transaction in transactions {
+                    self.send_over_bridge(BusMsg::Ctl(CtlMsg::TxFound(crate::bus::TxStatus {
+                        txid: transaction.txid(),
+                        block_pos: None,
+                    })))
+                    .expect("unable forward electrum notifications over the bridge");
+                }
+            }
+            _ => { /* nothing to do here */ }
```

Comment:
When we will subscribe for more event types this will cause a bug of losing notifications

Reply:
Before writing a message converter we can just enumerate all `ElectrumUpdate` variants here via `|` and leave this `{ /* notinig to do ...*/ }`

When we add another electrum update in the future compiler will notify us about not all enum options covered in the match and we will not forget to add forwarding if needed

Reply:
Done.

I will try to publish the converter together with the PR of interop with other LN implementations.